x-pack/filebeat/module/rabbitmq/log/ingest/pipeline.yml

```diff
@@ -0,0 +1,27 @@
+---
+description: Pipeline for parsing RabbitMQ logs
+processors:
+- grok:
+    field: message
+    pattern_definitions:
+      GREEDYMULTILINE: "(.|\n)*"
+      ERL_PID: "\\<%{INT}+\\.%{INT}+\\.%{INT}+\\>"
+    patterns:
+    - "%{DATESTAMP:timestamp} \\[%{WORD:log.level}\\] %{ERL_PID:rabbitmq.log.pid}
+      %{GREEDYMULTILINE:message}"
+    ignore_missing: true
+- date:
+    field: timestamp
+    formats:
+    - yy-MM-dd HH:mm:ss.SSS
+    ignore_failure: true
+    #{< if .convert_timezone >}
```

Comment:
As this module is only file based at the moment, should we skip the convert timezone completely and have it always enabled until requested otherwise by users?

Reply:
I figured I'd keep it to be consistent with the others and allow the user to disable it if they choose.

I think enabling it by default on this module makes sense b/c the [default log is localtime](https://github.com/elastic/beats/pull/12032#issuecomment-489758272) so I'm going to do that and make sure everything still works as expected.

Reply:
SGTM